src/ioctls/vcpu.rs

```diff
@@ -220,10 +220,9 @@ impl VcpuFd {
     /// let kvm = Kvm::new().unwrap();
     /// let vm = kvm.create_vm().unwrap();
     /// let vcpu = vm.create_vcpu(0).unwrap();
-    /// #[cfg(not(any(target_arch = "arm", target_arch = "aarch64")))]
     /// let regs = vcpu.get_regs().unwrap();
     /// ```
-    #[cfg(not(any(target_arch = "arm", target_arch = "aarch64")))]
+    #[cfg(not(any(target_arch = "arm", target_arch = "aarch64", target_arch = "riscv64")))]
```

Comment:
Let's hold off on adding the riscv64 directives until #275

Reply:
Noted 👍